db/migrate/20190916105800_create_zipcode_rates_archive.rb

```diff
@@ -0,0 +1,15 @@
+class CreateZipcodeRatesArchive < ActiveRecord::Migration
+  def up
+    safety_assured do
+      execute "create table zipcode_rate_archives (like zipcode_rates
```

Comment:
For the sake of consistency, can we call this table `zipcode_rates_archives`?
We archive institutions to `institutions_archives`, not `institution_archives`.